kandinsky/Makefile

```diff
@@ -66,38 +66,53 @@ ifdef HAS_READER
 
 kandinsky_src += $(addprefix kandinsky/fonts/, \
   LargeFontExtended.ttf \
+  ItalicLargeFontExtended.ttf \
   SmallFontExtended.ttf \
+  ItalicSmallFontExtended.ttf \
   LargeFontSimple.ttf \
+  ItalicLargeFontSimple.ttf \
   SmallFontSimple.ttf \
+  ItalicSmallFontSimple.ttf \
```

Comment:
Does the italic simple font has a utility ?

Reply:
In fact I just put it there because I don't really get what the Simple and Extended are for.
The Italic Font is used only in the code and graph app.

Reply:
The simple font is used for the flasher, it contains only the necessary code points, because the memory is limited

Reply:
ok thanks for the clarification